File: BSDN-API/src/main/java/com/gxu/bsdn/entity/Comment.java
package com.gxu.bsdn.entity;


import java.io.Serializable;
import java.util.Date;
import lombok.Data;


@Data
public class Comment implements Serializable {

    private Long id;


    private String content;


    private Long userId;


    private Long articleId;


    private Date createTime;

    private static final long serialVersionUID = 1L;
}

File: BSDN-Admin/vite-vue-admin/src/api/table/types/comment.ts
import {User} from "@/api/table/types/user";


export interface Comment {
  id: number,
  content: string,
  userId: number,
  articleId: number,
  createTime: string
  user: User
}
export interface CommentParam {
  userId: number,
  articleId: number
}
